### ISSUE TITLE
Add Archive web form of page.

### ISSUE THREAD

**Issue by ktorbeck** [OpenMediaVault-Plugin-Developers/openmediavault-borgbackup#15]
I am try to add an archive, but I can't because the top of the web form is not visible. I have tried Chrome, Classic Edge, and if the new version of Edge and I am still having the same problem. I resize the window, zoomed in and out, and even change the font size.
![Annotation 2020-07-17 121817](https://user-images.githubusercontent.com/6749800/87813813-37490200-c828-11ea-942c-44cb88b139d9.png)
![Annotation 2020-07-17 121818](https://user-images.githubusercontent.com/6749800/87813816-37e19880-c828-11ea-812b-23ad71bb20ab.png)

**Comment by ryecoaaron** [comment]
What resolution is your screen?  And when you resize the window, it should add a scrollbar.  If this isn't happening, it has something to do with your browser.  I just tried Chrome and Firefox with no issues.

**Comment by ktorbeck** [comment]
> What resolution is your screen?
I have tried do different laptops. One running at 3840x2160 and the other 1920x1080

> And when you resize the window, it should add a scrollbar. 
I get the scrollbar, but it will not scroll all the way to the top

![addarchiveerror](https://user-images.githubusercontent.com/6749800/87817371-711d0700-c82e-11ea-8ada-c2022e9f7e46.gif)

Google Photos Link to video showing error. 
https://photos.app.goo.gl/BZTmYCSooBtMxdgS9

**Comment by ryecoaaron** [comment]
Do you have the site zoomed in?  Try clearing your browser cache or the Reset UI to Defaults option in the OMV menu in the top right of the screen.

**Comment by ktorbeck** [comment]
> Do you have the site zoomed in? 
In the video and animated gif I sent, I started at 100% and zoomed out. to about 70%. My default is 100%. 

> Try clearing your browser cache or the Reset UI to Defaults option in the OMV menu in the top right of the screen.
- I will try clearing the browser cache, but the one laptop I got yesterday and it has only accessed OMV. Plus I had the issue on two different computers both on Chrome and Edge so I am not sure that would be the issue.
- I Reset the Ui and now I can see the top of the form. Zoomed out and resized it and now I can scroll to access all fields. 

I wonder if a two column form would help prevent this issue for others in the future.
